src/promptflow-devkit/promptflow/_sdk/_trace_telemetry.py

```diff
@@ -0,0 +1,36 @@
+import typing
```

Comment:
how about moving this to `/src/promptflow-devkit/promptflow/_sdk/_utilities/tracing_utils.py`? which I think is a better location to place util funcs